bgpd/bgp_evpn_vty.c

```diff
@@ -475,8 +482,30 @@ static void display_vni(struct vty *vty, struct bgpevpn *vpn, json_object *json)
 				       inet_ntoa(vpn->originator_ip));
 		json_object_string_add(json, "mcastGroup",
 				inet_ntoa(vpn->mcast_grp));
-		json_object_string_add(json, "advertiseGatewayMacip",
-				       vpn->advertise_gw_macip ? "Yes" : "No");
+		/* per vni knob is enabled -- Enabled
+		 * Global knob is enabled  -- Active
+		 * default  -- Disabled
```

Comment:
can't this be enabled both globally and per VNI at the same time?

Reply:
Yes It can be enabled in that case per vni knob will display from ``Active` to `Enabled`